internal/modifiers/timemodifier/replace_if_outdated.go

```diff
@@ -0,0 +1,66 @@
+package timemodifier
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func ReplaceIfOutdated() tfsdk.AttributePlanModifier {
+	return RequiresReplaceModifier{}
+}
+
+// RequiresReplaceModifier is an AttributePlanModifier that sets RequiresReplace
+// on the attribute.
+type RequiresReplaceModifier struct{}
```

Comment:
Is this accurate? For reference:

- https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/resource#RequiresReplaceIfFunc
- https://github.com/hashicorp/terraform-plugin-framework/blob/56ca8c935b997cec0a1a9441cc6b5a7cea4fcd2a/resource/plan_modifiers.go#L217

```suggestion
// RequiresReplaceModifier is an AttributePlanModifier that sets RequiresReplace
// on the attribute if the current time is past the stored timestamp.
//
// This custom modifier is necessary because the resource.RequiresReplaceIf
// function uses special logic for Computed attributes which is not applicable
// this use case.
type RequiresReplaceModifier struct{}
```